src/server/scripts/Commands/cs_server.cpp

```diff
@@ -132,7 +132,10 @@ class server_commandscript : public CommandScript
     // Display the 'Message of the day' for the realm
     static bool HandleServerMotdCommand(ChatHandler* handler, char const* /*args*/)
     {
-        handler->PSendSysMessage(LANG_MOTD_CURRENT, Motd::GetMotd());
+        if (handler->IsHumanReadable())
```

Comment:
Eh, main issue I have with that one is addons having to rely on the output order (since we wouldn't send any sort of identifying tag, e.g. `guid: whatever` or `npcflag: stuff`)